<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Converter</class>
 <widget class="QMainWindow" name="Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Converter</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: rgb(216, 255, 216);  /* Бледно-зелёный фон */
}

QPushButton {
    background-color: #FFD8A6;            /* Бледно-оранжевый */
    color: #5A4A3A;                      /* Тёмно-бежевый текст */
    border: 1px solid #E0B080;           /* Граница оранжевого оттенка */
    border-radius: 5px;
    font-size: 16px;
    min-width: 40px;
    min-height: 40px;
    font-weight: 500;                    /* Полужирный текст */
}

QPushButton:hover {
    background-color: #FFC080;           /* Более насыщенный оранжевый */
    border: 1px solid #D0A070;
}

QPushButton:pressed {
    background-color: #FFB366;           /* Ещё более тёмный оранжевый */
    border: 1px solid #C09060;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="conversionType">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="fromUnit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="toUnit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="resultValue">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите число в поле выше</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>270</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Перевести</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>382</x>
      <y>270</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернуться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
